sig-platform-dev/README.md

```diff
@@ -0,0 +1,83 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+-->
+# Platform Dev Special Interest Group
+
+SIG Platform Dev (SIG-PD) is a cross-cutting SIG whose mission is to design, build, and maintain a cohesive, first-party Platform Development Kit (PDK) for developers who want to extend the Kubernetes platform by adding new APIs or enhancing existing APIs, as well as to advocate on behalf of such developers by driving feedback into other SIGs.
+
+## Meetings
+* Regular SIG Meeting: [Tuesdays at 10:00 PT (Pacific Time)]() (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+
+* Jimmy Zelinskie (**[@jzelinskie](https://github.com/jzelinskie)**), Red Hat
+* Anthony Yeh (**[@enisoc](https://github.com/enisoc)**), Google
+
+### Technical Leads
+The Technical Leads of the SIG establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Jimmy Zelinskie (**[@jzelinskie](https://github.com/jzelinskie)**), Red Hat
+* Anthony Yeh (**[@enisoc](https://github.com/enisoc)**), Google
+* Phillip Wittrock (**[@pwittrock](https://github.com/pwittrock)**), Google
+
+## Contact
+* [Slack](https://kubernetes.slack.com/messages/sig-platform-dev)
+* [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-platform-dev)
+* [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fplatform-dev)
+
+## Subprojects
+
+The following subprojects are owned by sig-platform-dev:
+- **pdk-roadmap**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/pdk-roadmap/master/OWNERS
+- **sample-apiserver**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/sample-apiserver/master/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/staging/src/k8s.io/sample-apiserver/master/OWNERS
+- **sample-controller**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/sample-controller/master/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/staging/src/k8s.io/sample-controller/master/OWNERS
+- **gengo**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/gengo/master/OWNERS
+- **code-generator**
```

Comment:
Where does kube-openapi fall within this charter?

As much as it tries to be a general purpose framework/library, it is closely tied to gengo and what with the proliferation of API extensions, will become more important as time goes on (wrt automatic generation of api validation schemas, or docs). Would this fall within sig-pd?

Reply:
I think it is in the same class as k8s.io/code-generator: a foundational tool deeply linked to k8s.io/apiserver.

I would use the term "integrational tools" vs. "foundational tools" as the rule of thumb between PD and AM. This clarification does not mean that we cannot improve the user-experience of these tools and to broaden its purpose for 3rdparty developers.